internal/csi-addons/rbd/volumegroup.go

```diff
@@ -192,6 +194,12 @@ func (vs *VolumeGroupServer) DeleteVolumeGroup(
 	// resolve the volume group
 	vg, err := mgr.GetVolumeGroupByID(ctx, req.GetVolumeGroupId())
 	if err != nil {
+		if errors.Is(err, group.ErrRBDGroupNotFound) {
```

Comment:
same error check needs to be added for `/ControllerGetVolumeGroup`?

Reply:
We are already returning `NotFound` status code in case of an error there, so no update needed for ControllerGetVolumeGroup RPC.

Reply:
If not found it should be a success response.

Reply:
In ControllerGetVolumeGroup we should return success on not found, we should return NotFound only, right?
Else, how would we know that the volume group doesn't exists?

Reply:
Yes correct in ControllerGetVolumeGroup its notFound error but in DeleteVolumeGroup it should be success, this current change is for DeleteVolumeGroup RPC call isnt it?

Reply:
Yes, and we are returning a success reponse only.
`return nil, status.Errorf(codes.OK, "volume group %q doesn't exists", req.GetVolumeGroupId())`
The `codes.OK` basically, leads to a `nil` error only.

Reply:
Yes but we need to follow similar flow what we are following for other Delete operation, we return emptyResp,nil for better readability `&volumegroup.DeleteVolumeGroupResponse{}, nil` like its returned at the end.

Reply:
Ack, I will update it to return an empty response instead.

Reply:
Done